react-components/src/notifications/view/NotificationView.js

```diff
@@ -259,10 +258,9 @@ class NotificationView extends Component {
                                                  seen={n.seen}
                                                  presenter={this.props.presenter}
                                                  classes={classes}/>
-                                        <TableCell padding="none">{(n.message.timestamp) ?
-                                            moment(n.message.timestamp, "x").format(
-                                                constants.DATE_FORMAT) :
-                                            getMessage("emptyValue")} </TableCell>
+                                        <TableCell padding="none">
+                                            {formatDate(n.message.timestamp)}
```

Comment:
I like that you're updating other places where `formatDate` can be used! 👍 

This format used to use `constants.DATE_FORMAT`, but the default `format` parameter for `formatDate` is `constants.LONG_DATE_FORMAT`.
Should this be updated to `formatDate(n.message.timestamp, constants.DATE_FORMAT)`?